src/segments/lastfm.go

```diff
@@ -0,0 +1,159 @@
+package segments
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+
+	"github.com/jandedobbeleer/oh-my-posh/src/platform"
+	"github.com/jandedobbeleer/oh-my-posh/src/properties"
+)
+
+type LastFM struct {
+	props properties.Properties
+	env   platform.Environment
+
+	Artist string
+	Track  string
+	Full   string
+	URL    string
+	Icon   string
+	Status string
+}
+
+const (
+	// APIKey openweathermap api key
+	lfmAPIKey properties.Property = "apiKey"
+	// LastFM username
+	lfmUsername properties.Property = "username"
+	// Seconds since last played until the song is considered stopped
+	/* lfmStoppedSeconds properties.Property = "stopped_seconds" */ // Commented out incase we want to re-enable this feature
+	// CacheKeyResponse key used when caching the response
+	lfmCacheKeyResponse string = "lfm_response"
+	// CacheKeyURL key used when caching the url responsible for the response
+	lfmCacheKeyURL string = "lfm_url"
+
+	// PlayingIcon indicates a song is playing
+	lfmPlayingIcon properties.Property = "playing_icon"
```

Comment:
The spotify segment already has these as `PlayingIcon` and `StoppedIcon` so you can remove these and reuse those.